### DISCUSSION TITLE
사진찍기 UI 질문?

### DISCUSSION THREAD

**Discussion by peterkangdev** [naverz/zepeto-studio-kor#93] [5. Q&A]
등록된 월드에 들어가보면 사진찍기 라던지  여러가지 설정이 있는데, 그건 수동으로 만들어 줘야 하나요?
만약 수동으로 만드려면 어떻게 해야 하는지 알려주시면 감사하겠습니다.

**Comment by x4cf** [comment]
스크린샷 버튼은 아직 제작하실 수 없습니다.

**Comment by peterkangdev** [comment]
답변 감사하구요.. 한가지 질문 더 드리겠습니다. 

그럼 제페토 월드에서 일반 사용자들이 만든 월드에서 보이는 사진찍기 등은 어디서 만든 건가요?

**Reply by x4cf** [reply]
제가알기로 월드에서는 스크린샷은 구현 불가능한 기능입니다. 월드중에 스크린샷 기능이 있는 월드가있는걸까요?

**Comment by hyeonjin-jo** [comment]
안녕하세요.

현재 크리에이터에게 오픈한 기능 중에 사진 찍기 기능은 없는 점 참고 부탁드립니다.

**Comment by strangestraw** [comment]
안녕하세요. 혹시 Unity 상에서 스크린샷과 동영상 저장 기능을 구현해서 화면을 녹화하는 것은 가능한가요?
Unity 기술적으로는 문제 없는 것으로 알고 있는데, 제페토 정책 / TS 지원에는 문제 없는 지 궁금합니다.

**Reply by x4cf** [reply]
불가능합니다.
제페토 API를 이용하여 유니티 안에서 활용하는건 기술적으로는 문제가 없습니다.
하지만 유니티와 같은 엔진서비스가 아닌 플렛폼서비스인 제페토에 File I/O 기능이 완전히 오픈된 어플리케이션을 업로드하는건 어려워보입니다.

**Reply by x4cf** [reply]
사진찍기나 녹화기능, 통화기능은 이후에 업데이트로 기능사용이 풀릴 수 있을지도 모르지만 제페토 월드 1.1.2 에서는 불가능합니다.
